src/components/Footer/Footer.tsx

```diff
@@ -1,24 +1,33 @@
 import React from 'react';
 
+import FooterMain from './FooterMain/FooterMain';
 import FooterSocial from './FooterSocial/FooterSocial';
 import FooterLinks from './FooterLinks/FooterLinks';
 
 type FooterProps = {
   links?: [];
+  mainLinks?: [];
   socialLinks?: [];
 };
 
-export const Footer = ({ links, socialLinks }: FooterProps) => (
-  <footer className="footer">
-    <div className="footer__container">
-      <h3 className="footer__mission-statement">
-        Wellcome exists to improve health by helping great ideas to thrive
-      </h3>
-      {/* @todo: #5988 add FooterNav component */}
-      <FooterSocial links={socialLinks} />
-      <FooterLinks links={links} />
-    </div>
-  </footer>
-);
+export const Footer = ({ links, mainLinks, socialLinks }: FooterProps) => {
+  return (
+    <footer className="footer">
+      <div className="footer__container">
+        <h3 className="footer__mission-statement">
+          Wellcome exists to improve health by helping great ideas to thrive
+        </h3>
+        <nav id="footer-nav" className="footer-nav">
+          <ul className="footer-nav__list footer-nav__list--grid">
+            <FooterMain mainLinks={mainLinks} />
```

Comment:
if the data prop for FooterSocial and FooterLinks is called **links** this one should probably follow suit, just to be consistent